### ISSUE TITLE
Entire list of blocks [95%] ExtendedBlocks

### ISSUE THREAD

**Issue by MrBlasyMSK** [xSuperr/VanillaBlocks#43]
I am going to share the list that I made during 3 days of all the blocks that I could find
Notice! All the names are in Spanish (because I'm Spanish)
AND NOT ALL BLOCKS HAVE THEIR CORRECT VALUES, IE: SILKTOUCK, FUELTIME ETC
(The IDs if they are correct) 
Blocks from 1.17 not added
I share this list so that you can get more IDs of objects (such as dead corals) which are the ids -134, -135, -136 (and the list does not contain the subsIDs -135: 1, -135: 2 etc but they work)

[BlocksIDlist.txt](https://github.com/xSuperr/VanillaBlocks/files/6836434/BlocksIDlist.txt)

**Comment by xSuperr** [comment]
While I do appreciate this, I would rather have a list of block ids that could be added to the plugin (Just the ids). I am capable to do research on the break speed, ect. Having such a large file with that much info makes it hard to look through. For now I'll leave this issue open to add my own version of that list later.

**Comment by MrBlasyMSK** [comment]
ID: 258 = -3 Dark Prismarine Stairs
ID: 266 = -11 Blue ice
ID: 385 = -130 Sea bottom
ID: 393 = -138 Alga
ID: 394 = -139 Block of dried seaweed
ID: 395 = -140 Acacia button
ID: 396 = -141 Abedul button
ID: 397 = -142 Dark oak button
ID: 398 = -143 Jungle button
ID: 399 = -144 Fir button
ID: 405 = -150 Acacia pressure plate
ID: 406 = -151 Birch pressure plate
ID: 407 = -152 Dark oak pressure plate
ID: 408 = -153 Jungle pressure plate
ID: 409 = -154 Fir pressure plate
ID: 410 = -155 Carved pumpkin
ID: 412 = -157 Conduit
ID: 414 = -159 Sea turtle egg
ID: 415 = -160 Bubble column
ID: 417 = -162 End Stone Brick Slab
ID: 418 = -163 Bamboo
ID: 419 = -164 Bamboo root
ID: 421 = -166 Mossy stone brick slab
ID: 433 = -178 End Stone Brick Stairs
ID: 436 = -181 Fir sign
ID: 438 = -183 Smooth stone
ID: 441 = -186 Birch sign
ID: 443 = -188 Jungle sign
ID: 445 = -190 Acacia sign
ID: 447 = -192 Dark oak sign
ID: 449 = -194 Lectern
ID: 450 = -195 Whetstone
ID: 451 = -196 Melting oven (469 = -214 Burning up)
ID: 452 = -197 Stone cutter
ID: 453 = -198 Smoker (454 = -199 Burning up)
ID: 455 = -200 Cartography table
ID: 456 = -201 Feathered table
ID: 457 = -202 Blacksmith table
ID: 459 = -204 Loom
ID: 461 = -206 Bell
ID: 462 = -207 Sweet berry bush
ID: 466 = -211 Puzzle block
ID: 467 = -212 Oak wood
ID: 468 = -213 Composter
ID: 470 = -215 Light block (I think I remember that this "block" needs subsIDs, and they were from 0 to 15 lighting)
ID: 471 = -216 Wither rose
ID: 473 = -218 Bee's nest
ID: 474 = -219 Hive
ID: 475 = -220 Honey block
ID: 476 = -221 Honeycomb block
ID: 477 = -222 Magnetite
ID: 478 = -223 Crimson roots
ID: 479 = -224 Deformed roots
ID: 480 = -225 Crimson stem
ID: 481 = -226 Deformed stem
ID: 482 = -227 Deformed wart block
ID: 483 = -228 Crimson mushrooms
ID: 484 = -229 Deformed mushrooms
ID: 485 = -230 Light mushroom
ID: 486 = -231 Weeping creepers
ID: 487 = -232 Crimson nylium
ID: 488 = -233 Deformed nylium
ID: 493 = -238 Outbreaks of the underworld
ID: 494 = -239 Objective
ID: 506 = -251 Warped signal
ID: 511 = -256 Crimson fence
ID: 512 = -257 Warped fence
ID: 513 = -258 Crimson fence gate
ID: 514 = -259 Warped fence gate
ID: 515 = -260 Crimson button
ID: 516 = -261 Deformed button
ID: 517 = -262 Crimson pressure plate
ID: 518 = -263 Deformed pressure plate
ID: 519 = -264 Crimson slab
ID: 520 = -265 Deformed slab
ID: 525 = -270 Inframundite Block
ID: 527 = -272 Regeneration Anchor
ID: 531 = -276 Blackrock stairs
ID: 536 = -281 Golden Blackrock
ID: 537 = -282 Blackrock Slab
ID: 539 = -284 Polished Blackrock Brick Slab
ID: 541 = -286 Chain
ID: 542 = -287 Coiled vines
ID: 544 = -289 Tearful obsidian
ID: 548 = -293 Polished Blackrock Slab
ID: 550 = -295 Polished black rock pressure plate
ID: 551 = -296 Polished Blackrock Button
ID: 553 = -298 Deformed hyphae
ID: 555 = -300 Crimson hyphae without bark
ID: 556 = -301 Deformed hyphae without bark
ID: 557 = -302 Chiseled Bricks of the Underworld
ID: 558 = -303 Cracked Underworld Bricks
ID: 559 = -304 Quartz bricks

I've spent 2 hours for your pretty ass, you better add them :)

**Comment by Nya-Enzo** [comment]
He could already get a sort of list on https://github.com/pmmp/BedrockData

**Comment by MrBlasyMSK** [comment]
Yes, but from that list there are many blocks that do not work, what I have done is to put them 1x1, in addition to that I have completed in my last comment all the blocks that are still to be put

> He could already get a sort of list on https://github.com/pmmp/BedrockData

**Comment by Nya-Enzo** [comment]
Oh, i didnt come to think of that. Also for the education blocks, you could handle the StartGamePacket and set education features to true, and eduOffer to EducationOffer::EVERYWHERE_EXCEPT_CHINA (didnt test it)
